src/app/components/LocaleSwitcher/LocaleSwitcher.tsx

```diff
@@ -0,0 +1,25 @@
+import { useState, ChangeEvent } from "react";
```

Comment:
Separate these into `import ...` and `import type ...`

Reply:
done